examples/ppo_sentiments.py

```diff
@@ -17,7 +17,7 @@ def get_positive_score(scores):
     return dict(map(lambda x: tuple(x.values()), scores))["POSITIVE"]
 
 
-default_config = yaml.safe_load(open("configs/ppo_config.yml"))
+default_config = yaml.safe_load(open("configs/ppo_gptj.yml"))
```

Comment:
are you sure you want to change the default to gpt-j? I wouldn't mind the second script which imports `main` from here and changes config as the most simplest option right now, until we switch to hydra or something else

Reply:
Agreed, this is a bad idea.

Reply:
Yeah I just forgot to change this back.